src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -90,18 +90,17 @@ export function CurrencySearch({
   const sortedTokens: Token[] = useMemo(
     () =>
       !balancesAreLoading
-        ? [...filteredTokens]
+        ? filteredTokens
             .filter((token) => {
```

Comment:
`filter` returns a shallow copy of the array, so it is not necessary to first make our own shallow copy (eg `[...filteredTokens]`)